### ISSUE TITLE
user survey questions

### ISSUE THREAD

**Issue by errordeveloper** [kubernetes/minikube#2525]
It'd be good to create a survey, so that we have a better idea of how folks use minikube and why they use it that way.

One of the key questions would be about what drivers folks use, and why that driver is important to them, so that we can consider what drivers can be safely deprecated, and find out what alternative options can serve the same use case.

This issue is for the discussion of what questions we want to ask the users.

**Comment by afbjorklund** [comment]
What bootstrapper that people use, could also be of interest. And the Kubernetes version, I suppose.

**Comment by errordeveloper** [comment]
>  And the Kubernetes version, I suppose.

Yes, I think we should have multiple questions about version, namely to identify how many folks would be okay with latest stable release and how many folks need some particular versions for some specific use-case.

**Comment by dlorenc** [comment]
How's this for a draft list:

1. What driver(s) do you use regularly
2. What platform do you run on (Windows, OSX, Linux)
3. What Kubernetes versions do you use
4. What --extra-config parameters do you use
5. What addons do you use

**Comment by dlorenc** [comment]
cc @kimsterv

**Comment by afbjorklund** [comment]
Something semi-related, that is: more of a future possibility, but still interesting to ask to ask users, would be on the lines of:
- are you interested in having a multi-node minikube (beyond current single-node)

**Comment by errordeveloper** [comment]
From [wg-app-def survey](https://groups.google.com/d/msg/kubernetes-sig-apps/0t3I0O1Jvic/SDzbAv7tBgAJ):

- How would you describe your usage of Minikube 
  - I regularly use Minikube
  - I use a competitor to Minikube instead
  - I do not use Minikube or any of its competitors
- What do you like about Minikube? (if they use minikube)
- What would you like to see added or change in Minikube? (if they use minikube)
- Which competitor to Minikube do you use and why do you use it? (if they use a competitor)
- What platform do you run on (Windows, OSX, Linux)
- What driver(s) do you use regularly with Minikube?
- What bootstrapper do you use with Minikube?
- What version of Kubernetes are up you using? Do you need support for older version or is the latest stable release enough?
- What --extra-config parameters do you use?
- What addons do you use?
- Are you interested in having a multi-node minikube (beyond current single-node)?

**Comment by errordeveloper** [comment]
Do we think we have enough here to sketch a basic survey form or there is anything we should add?

I think wg-app-def has a good number of questions, but participants have to answer more questions, and it seems like a rather better option to have more short and targeted surveys... so perhaps we could ask wg-app-def to relay some of the deeper questions onto this survey?

**Comment by r2d4** [comment]
Did this go out already? I changed up some of the wording to not be so harsh (alternative rather than competitor) and added the question about container-runtime

**Comment by errordeveloper** [comment]
@r2d4 no, I don't think it went out yet. Last time I checked with @mattfarina, he said they were going to come back to it after Helm Summit (which was just over a week ago).
